packages/x-adapter-platform/src/types/request.types.ts

```diff
@@ -1,6 +1,3 @@
-import { Filter } from '@empathyco/x-types';
-import { Dictionary } from '@empathyco/x-utils';
-
 export interface BaseRequest {
```

Comment:
should we move this to a base-request file?

Reply:
will be used when we finish this refactor? I thought we are getting rid of this BaseRequest after all.